### ISSUE TITLE
"Tab" key (or mobile Up/Down arrows) inside large sections

### ISSUE THREAD

**Issue by Khusainov** [alvarotrigo/fullPage.js#3790]
### Description
Navigation between inputs inside large section by using Tab key (or Up/Down buttons on iOS) brokes the fullpage navigation.

### Link to isolated reproduction with no external CSS / JS
https://jsfiddle.net/damirk/ofqk9yxt/9/

### Steps to reproduce it
1. Try to navigate between sections. Everything works fine (3 sections in example)
2. Go to section 2 and click on first text input
3. Press Tab button unless you get last input (datetime input)
4. Try to navigate between sections again. Now you cannot open first section. When you go to the last section you get white screen.

### Versions 
Tested on Chrome, Win10
iOS (iPhone XS)

Similar to the issue #1237 but inside one section

**Comment by alvarotrigo** [comment]
Do you get the same issue when using the `scrollOverflow` option so it will create a scrollbar on the bigger sections?

**Comment by Khusainov** [comment]
Yes, same issue occurs with `scrollOverflow` option too:
https://jsfiddle.net/damirk/ofqk9yxt/11/

**Comment by alvarotrigo** [comment]
> (or Up/Down buttons on iOS)

What do you mean by that?
What are the up / down arrows in iOS?

**Comment by Khusainov** [comment]
I meant following buttons in Safari iOS:
![image](https://user-images.githubusercontent.com/367404/68693600-a16a2280-0588-11ea-9329-38eb60f9244a.png)

**Comment by alvarotrigo** [comment]
Thanks for that! 
7 years using and iPhone and never used those or even realise they were there! 😆 

The issue doesn't sound like a simple one! But I'll take a look at it these days!

**Comment by alvarotrigo** [comment]
> Navigation between inputs inside large section by using Tab key (or Up/Down buttons on iOS) brokes the fullpage navigation.

I'm wondering... why would you have a long section when not using scrollOverflow?
Were you using `autoScrolling: false`? Or you were using `scrollOverflow` but just omitted  it on the question?

**Comment by Khusainov** [comment]
Your assumption is correct. We are using scrollOverflow and it was omitted on the question. Just want to show you that bug reproduced independently of scrollOverflow using.

Regards,
Damir

> 21 нояб. 2019 г., в 19:39, Álvaro <notifications@github.com> написал(а):
> 
> were using scrollOverflow but just omitted it on the question?

**Comment by alvarotrigo** [comment]
> Just want to show you that bug reproduced independently of scrollOverflow using.

Yeah, just mentioning it because scrollOverflow adds another layer of complexity to the issue :)

**Comment by alvarotrigo** [comment]
The issue also exist on desktop, so it is not only a mobile issue.
I've managed to find what seems to be a solution for desktop, but  not yet for Safari. In fact, I can't even trigger the focus event there...

Perhaps it is not even possible after all?
![image](https://user-images.githubusercontent.com/1706326/69369183-e0d8f300-0c9b-11ea-8e10-2f3696f44f52.png)
Source:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus
